render-configs.sh

```diff
@@ -19,9 +19,12 @@
 # Then, refresh your z-shell configuration (`source ~./zshrc`)
 # ./gradlew bootRun
 
+set -eu
+
 AZURE_ENV=dev
 RBS_ENV=tools
 COPY_INTELLIJ_ENV_VARS=n
+USE_VAULT="${USE_VAULT:-false}"
```

Comment:
I like leaving this backdoor to Vault for the time being.  Should we also include it in the script documentation above?

Better yet, I think this script would also benefit from a usage command like you added for the DB connect script.  But it would be fair to call that improvement OOS.

Reply:
I initially had this in, but I think we should not encourage the use of Vault through a CLI argument, hence why I removed it. Relatedly my idea was once this change was merged and we were comfortable with GSM then I would make a separate PR that deprecates Vault completely and removes this and the other codepaths.

Reply:
I created https://broadworkbench.atlassian.net/browse/DCJ-526 to address these items.